defender/staged/downloaded/Forta_Governance.spec.js

```diff
@@ -45,7 +45,7 @@ const mockExecutedMeta = {
   state: 'executed',
```

Comment:
Several of these tests need to have `axios.get` mocked as well, so that the Compound API call is mocked appropriately and the proposal name can be returned.

Reply:
fixed in commit 846f1abf484dcd789d1eea6473ba71be8b1b18e3